### DISCUSSION TITLE
getting this error on dev

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#427] [! Welcome - readme first!]
When I click on these to add a new op i get this error in all my patches.
<img width="1088" alt="Screenshot 2022-12-11 at 17 04 50" src="https://user-images.githubusercontent.com/36418408/206917861-1be73392-4373-4851-a58e-fb7d9d170957.png">

**Comment by pandrr** [comment]
thanks, should be fixed on dev.cables.gl now
